Examples/Scripts/Python/full_chain_itk_FTF.py

```diff
@@ -78,7 +78,9 @@
     field,
     digiConfigFile=geo_dir
     / "itk-hgtd/itk-smearing-config.json",  # change this file to make it do digitization
-    outputDirRoot=outputDir,
+    # digiConfigFile=geo_dir
+    # / "itk-hgtd/itk-pixel-digitization.json",
+    # outputDirRoot=outputDir,
```

Comment:
are these changes in `full_chain_itk_FTF.py` intended to be merged?

Reply:
good catch :)

Reply:
No sorry, thank you for catching this!